pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md

```diff
@@ -54,6 +54,17 @@ gcloud container clusters list
 export GCP_PROJECT=$(gcloud config list --format='value(core.project)')
 ```
 
+## Retrieve the Cluster credentials
+
+```
+gcloud config set container/use_client_certificate True
+export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
```

Comment:
Indeed, I was able to remove the get-credentials part by moving use_client_certificate before the cluster create commands.